pupil-spa/src/app/services/submission/submission.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { Http, RequestOptions, Headers } from '@angular/http';
+// import { environment } from '../../../environments/environment';
+import { StorageService } from '../storage/storage.service';
+
+@Injectable()
+export class SubmissionService {
+
+  constructor(private http: Http, private storageService: StorageService) {
+  }
+
+  async submitData() {
+    const localStorageData = this.storageService.getAllItems();
+    const headers = new Headers();
+    headers.append('Content-Type', 'application/json');
+    const requestArgs = new RequestOptions({headers: headers});
+    // TODO: build Check complete API call
```

Comment:
Will be uncommented once the api call exists and tests will be amended/improved accordingly